clients/devtools/devtools-entry.js

```diff
@@ -68,6 +68,7 @@ function setUpWorkerConnection(port) {
 /** @param {(status: [string, string, string]) => void} listenCallback */
 function listenForStatus(listenCallback) {
   log.events.addListener('status', listenCallback);
+  log.events.addListener('warning', listenCallback);
```

Comment:
i assume warnings are rare and this doesn't make things too chatty?

Reply:
yeah they're very rare.   `status` is definitely the chatty one.